hr_employee_age/models/hr_employee.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
```

Comment:
Please follow the import guideline: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports